server/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const axios = require('axios');
+const db = require('../database/index.js')
+
+const app = express();
+const port = 3000;
+
+app.use(express.static(__dirname + '/../client/dist'));
+app.listen(port, () => { console.log(`Now listening on ${port}`) });
+
+
+// Build route handlers
+app.get('/', (req, res) => {
```

Comment:
Everything looks good and clean. Just one comment here: you might want to consider using a different handler instead of '/' to differentiate between a home page and API call.

Reply:
Right on! I'll make that adjustment. Thanks